pytest_mpi/plugin.py

```diff
@@ -304,44 +303,34 @@ class MPIImplementation(enum.Enum):
 
 
 def detect_mpi_implementation() -> MPIImplementation:
-    result = None
+    from mpi4py import rc
 
-    try:
-        result = subprocess.run(
-            ["mpiexec", "--version"],
-            stdout=subprocess.PIPE,
-            stderr=subprocess.STDOUT,
-            text=True,
-            check=True
-        )
-    except CalledProcessError:
-        pass
-
-    # MSMPI can be detected by just calling 'mpiexec'
-    if result is None and sys.platform.casefold().startswith("win"):
-        try:
-            result = subprocess.run(
-                ["mpiexec"],
-                stdout=subprocess.PIPE,
-                stderr=subprocess.STDOUT,
-                text=True,
-                check=True
-            )
-        except CalledProcessError:
-            pass
+    rc.initialize = rc.finalize = False
```

Comment:
I don't think that this is necessary because we already call `from mpi4py import MPI` further up the call stack.